src/actions/actions.js

```diff
@@ -10,4 +10,9 @@ const removeBook = (book) => ({
   payload: book,
 });
 
-export { createBook, removeBook };
+const changeFilter = (category) => ({
+  type: actions.CHANGE_FILTER,
+  payload: category,
+});
+
+export { createBook, removeBook, changeFilter };
```

Comment:
- [ ] Kindly change the name of the file to index.js instead of the actions.js we have currently, let us keep in mind we have a directory called "actions" already. Index.js files provide easy entry points for components/actions and also simplify imports :+1:

Reply:
This change has been done perfectly 👍🏿